mlir/lib/Dialect/LLVMIR/IR/LLVMInlining.cpp

```diff
@@ -132,6 +132,38 @@ namespace {
 struct LLVMInlinerInterface : public DialectInlinerInterface {
   using DialectInlinerInterface::DialectInlinerInterface;
 
+  LLVMInlinerInterface(Dialect *dialect)
+      : DialectInlinerInterface(dialect),
+        // Cache sets of `StringAttr` for fast lookup in `isLegalToInline`.
+        disallowedArgumentAttrs({
```

Comment:
Would the initialization format a bit nicer in the body of the constructor?

Reply:
It would format a bit nicer, but then they would be able to be `const` 🙂